static/needs-flexibility.js

```diff
@@ -0,0 +1,20 @@
+// Light-weight IE detection http://stackoverflow.com/a/19999868/2687646
+function msieversion() {
+
+    var ua = window.navigator.userAgent;
+    var msie = ua.indexOf("MSIE ");
+
+    if (msie > -1 || !!navigator.userAgent.match(/Trident.*rv\:11\./))  // If Internet Explorer, return version number
+    {
+        return parseInt(ua.substring(msie + 5, ua.indexOf(".", msie)));
+    }
+    else  // If another browser, return 0
+    {
+        return 0;
+    }
+    
+}
+
+var v = msieversion();
+
+export default (v >= 9 && v <= 10);
```

Comment:
@justinbmeyer I downloaded a build of Modernizr that only includes flexbox detector. The only way to download it was minified so I ran it through JSNice: [gist](https://gist.github.com/imaustink/9e256d92935b315512b1000fcc51b7dd)